cup of join.py

```diff
@@ -0,0 +1,16 @@
+
+def join(*lists,sep='-'):
+    """
+    The function receives lists and joins them into one list
+    :param lists:lists to join
+    :param sep: Character to be placed between lists
+    :return: The list that created
```

Comment:
Its not just a new list, its The new merged list with "sep" seperating between the recieved lists.